graphql-server/go-server/main.go

```diff
@@ -98,6 +98,7 @@ func main() {
 	http.HandleFunc("/minio/get_multipart_url", minio.GetMultipartUploadURL)
 	http.HandleFunc("/minio/complete_multipart", minio.CompleteMultipart)
 	http.HandleFunc("/minio/update_chunk", minio.UpdateMultipart)
+	http.HandleFunc("/minio/delete_files", minio.DeleteFiles)
```

Comment:
do we have any authorization control of read/write MinIO files?
AFAIK, we just check the validation of oidc token.

Reply:
Here we have a issue to track this permission check enhancement https://github.com/kubeagi/arcadia/issues/251 
Now odic check on file operation is disabled.

Reply:
https://min.io/docs/minio/kubernetes/upstream/operations/external-iam/configure-openid-external-identity-management.html

Reply:
Another option:
1. Create a pair of access_key & access_secret for each bucket(namespace)
    - Only have read/write permission to that bucket
2. Use the above access_key & access_secret to create a datasource for each namespace

Then when user upload files ,it :
1. get the datasource  from the namespace
2. use the access_key & access_secret to upload files

Reply:
@nkwangleiGIT @0xff-dev

Reply:
Can we watch the namespace changes and create the relevant datasource which is marked as `system` ?

Reply:
yes, for ak/sk, we can create separate one for each namespace
and for file operation, maybe we can use noResourceUrl for RBAC if we have different context path for MinIO access.